tests/api/preprocessing/test_normalization.py

```diff
@@ -0,0 +1,23 @@
+from pathlib import Path
+
+import numpy as np
+from anndata import AnnData
+
+from ehrapy.api.preprocessing import Normalization
+
+CURRENT_DIR = Path(__file__).parent
+_TEST_PATH = f"{CURRENT_DIR}/test_preprocessing"
+
+
+class TestNormalization:
+    def test_identity(self):
```

Comment:
I would suggest to add a setup method which creates a couple of AnnData objects to use for testing purposes.
We should test 1 AnnData object which only has numericals and one which has a mix of numericals and categoricals which got encoded earlier. You could also think about a case where X still contains strings which did not get encoded yet. These should not be used for the normalization. Also, the information in `uns` about which columns are categoricals may not yet exist. You could attempt to figure out which columns are Strings and therefore categoricals and proceed as usual or throw an exception telling people to encode their data first. Happy with either solution.

See: https://github.com/theislab/ehrapy/blob/development/tests/api/tools/nlp/test_translator.py#L23